performance/packages.lock.json

```diff
@@ -2088,37 +2088,37 @@
       "microsoft.azure.webjobs.extensions.sql": {
         "type": "Project",
         "dependencies": {
-          "Microsoft.ApplicationInsights": "2.17.0",
-          "Microsoft.AspNetCore.Http": "2.1.22",
-          "Microsoft.Azure.WebJobs": "3.0.31",
-          "Microsoft.Data.SqlClient": "3.0.1",
-          "Newtonsoft.Json": "13.0.1",
-          "System.Runtime.Caching": "4.7.0",
-          "morelinq": "3.3.2"
+          "Microsoft.ApplicationInsights": "[2.17.0, )",
```

Comment:
Why are these changing? Undo them if they aren't necessary.

Reply:
dotnet restore updated them automatically, will check and undo them if unnecessary.